modules/installing-using-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-using-assisted-installer-oci_{context}"]
+= Using the Assisted Installer to install a cluster on OCI
+
+In {product-title} {product-version}, you can use the Assisted Installer to install a cluster on {oci-first} by using infrastructure that you provide.
+
+:FeatureName: Using the Assisted Installer to install an {product-title} cluster on OCI
+include::snippets/technology-preview.adoc[]
+
+You can access {oci} Resource Manager configurations to provision and configure {oci} resources that meet your organization's needs.
+
+[IMPORTANT]
+====
+The steps for provisioning {oci} resources are provided as an example only. You can also choose to create the required resources through other methods; the scripts are just an example. Installing a cluster with infrastructure that you provide requires knowledge of the cloud provider and the installation process on {product-title}. You can access {oci} Resource Manager configurations to complete these steps, or use the configurations to model your own custom script.
+====
+
+Follow the steps in the procedure to understand how to use the Assisted Installer to install a {product-title} cluster on OCI. The procedure demonstrates the use of the {oci}'s Manager (CCM) and Oracle’s Container Storage Interface (CSI) objects to link your {product-title} cluster with {oci}'s API.
```

Comment:
I noticed that using `}'` makes everything after these characters appear italicized. I think it is better to use the platform name as an adjective instead of using the possessive apostrophe. Long ago, we had a guideline not to use the possessive apostrophe but we dropped it. However, I think that avoiding it produces better documentation. 

- "OCI Manager" instead of "OCI's Manager"
- "with the OCI API" instead of "with OCI's API"